frontend/src/handlers/createProjectSubmitHandler.tsx

```diff
@@ -0,0 +1,69 @@
+import apiPaths from "../properties/apiPaths";
+import Router from "next/router";
+import axios from "axios";
+import { AxiosConf, AxiosFormConfig, ManyToManyAxiosConf } from "../api/calls/baseCalls";
+import { Project } from "../api/entities/ProjectEntity";
+import applicationPaths from "../properties/applicationPaths";
+import { ProjectSkill } from "../api/entities/ProjectSkillEntity";
+
+export interface ProjectCreationValues {
+    projectName: string;
+    projectInfo: string;
+    versionManagement: string;
+    partnerName: string;
+    partnerWebsite: string;
+    skills: string[];
+    skillInfos: string[];
+    coaches: string[];
+}
+
+export type ProjectCreationProps = {
+    submitHandler: (values: ProjectCreationValues) => void;
+};
+
+export async function createProjectSubmitHandler(values: ProjectCreationValues) {
+    const ownUser = await axios.get(apiPaths.ownUser, AxiosConf);
+    const project: Project = new Project(
+        values.projectName,
+        values.projectInfo,
+        values.versionManagement,
+        [],
+        values.partnerName,
+        values.partnerWebsite,
+        "/editions/3",
```

Comment:
Is this a hard coded edition? This should not be hard coded

Reply:
Jitse allowed this because there is no global edition state yet

Reply:
Only if you mention it in #346.